lambda/kendra-webcrawler-schedule-updater/kendra_webcrawler_schedule_updater.py

```diff
@@ -128,26 +133,8 @@ def kendra_update_data_source(IndexId, data_source_id, URLs, RoleArn, schedule,
                         'WebCrawlerMode': 'EVERYTHING'
                     }
                 },
-                'CrawlDepth': crawl_depth
+                'CrawlDepth': int(crawl_depth)
             }
         }
     )
     return response
-
-
-def create_dashboard(IndexId, data_source_id):
```

Comment:
There is still a call to create_dashboard from the kendra webcrawler handler when the data source id is None? Is this intentional?

Reply:
This was a duplicate definition and fixed now. Thanks.